tokio/src/io/util/repeat.rs

```diff
@@ -61,6 +63,20 @@ impl AsyncRead for Repeat {
     }
 }
 
+cfg_coop! {
+    fn poll_proceed_and_make_progress(cx: &mut Context<'_>) -> Poll<()> {
```

Comment:
I belive these codes are duplicates from file `tokio/src/io/util/empty.rs`, we can move these code to `tokio/src/io/util/mod.rs`.